### ISSUE TITLE
Switch to GPLv2

### ISSUE THREAD

**Issue by pachpict** [Bristol-Braille/canute-ui#166]
I've been told by a few screen reader developers that they are contractually
forbidden from contributing to GPLv3 projects, but GPLv2 is fine. It looks
like the FSF still approve the use of GPLv2;

"First of all, it is important to note that upgrading is a choice. GPL
version 2 will remain a valid license, and no disaster will happen if some
programs remain under GPLv2 while others advance to GPLv3."

In short, we can get contributions from more of our commercial partners 
if we switch to GPLv2, so we should do so.

**Comment by rbubley** [comment]
Just in case you need your contributors to consent to a change of license, you have mine.

**Comment by kasbah** [comment]
Two things to consider:

1. We currently make use of PyQt which is licensed GPLv3 so to continue to make use of that we need to stay GPLv3 compatible i.e. could only switch to "GPLv2 or any greater version" 
    - This only affects the emulator so could make a GPLv2 licensed version with this function disabled. 
    - We can also switch back to PySide a.k.a "Qt for Python" as I hear they have updated that and it's more permissively licensed (I previously switched away from it as it hadn't been updated for Python 3.6) 
2. One of the reasons for the GPLv3 was the emergence of [tivoization](https://en.wikipedia.org/wiki/Tivoization). You should consider whether it's a significant risk and/or potential problem that a competitor or partner makes modified versions of the canute-ui software and sells devices with it on it but doesn't release the source code for these modifications. 

I think with @rbubley chiming in we do have all copyright holders (i.e. BBT & @rbubley) agreement for a license change so just let me know where you come down on either: 

1. Stay GPLv3
2. Move to GPLv2+ but stay with PyQt (meaning effectively we are GPLv3+, note the `+`) 
3. Move to dual licensed: GPLv2 without emulator & GPLv3 with emulator (seems confusing)
4. Move to GPLv2 and switch to PySide (seems like wasted work and we could run into future licensing incompatibilities if we want to make use of GPLv3 licensed libraries)

**Comment by kasbah** [comment]
No, GPLv3+ since PyQt is GPLv3 so anyone making copying project when it includes the emulator is bound by GPLv3 and not at all by GPLv2.

**Comment by rbubley** [comment]
A variant of @kasbah’s option (3) is used by various other GPLv2 projects - e.g. ffmpeg:

If this approach were adopted, the project would be GPLv2, and would then need to be distributed by default without PyQT turned on, however the user would be able to specify an installation flag that installed PyQT and enabled the emulator — but this installed version would then not be redistributable.

**Comment by pachpict** [comment]
We should hear more from screen reader developers over the summer, so this is being added to the post-release milestone.
